gatsby-node.js

```diff
@@ -14,39 +13,41 @@ const path = require('path');
 const identity = obj => obj;
 
 exports.onPostBuild = async function(
-  { graphql },
+  { cache, graphql },
```

Comment:
great call finding that gatsby has an API for setting cache as a plugin, I wasn't aware of it yet!